kordac/tests/ConfigurationTest.py

```diff
@@ -26,6 +28,98 @@ def __init__(self, *args, **kwargs):
             "boxed-text": "<div class='box'>{% autoescape false %}{{ text }}{% endautoescape %}</div>"
         }
 
+    def test_multiple_calls(self):
+        """Checks all fields of KordacResult are correct for multiple Kordac calls"""
+        test_cases = [
+            ('all_processors.md',
+                KordacResult(
+                    html_string=self.read_test_file(self.test_name, 'all_processors_expected.html', strip=True),
+                    title='Example Title',
+                    required_files={
+                        'interactives': set(),
+                        'images': set(),
+                        'page_scripts': set(),
+                        'scratch_images': set()
+                    },
+                    heading_tree=(HeadingNode(
+                        title='Example Title',
+                        title_slug='example-title',
+                        level=1,
+                        children=()
+                    ),),
+                    required_glossary_terms=defaultdict(list)
+                )
+            ),
+            ('some_processors.md',
+                KordacResult(
+                    html_string=self.read_test_file(self.test_name, 'some_processors_expected.html', strip=True),
+                    title='Another Example Title',
+                    required_files={
+                        'interactives': set(),
+                        'images': {'totally-legit-image.png'},
+                        'page_scripts': set(),
+                        'scratch_images': set()
+                    },
+                    heading_tree=(HeadingNode(
+                        title='Another Example Title',
+                        title_slug='another-example-title',
+                        level=1,
+                        children=(HeadingNode(
+                            title='This is an H2',
+                            title_slug='this-is-an-h2',
+                            level=2,
+                            children=()
+                        ),),
+                    ),),
+                    required_glossary_terms={
+                        'chomsky-hierarchy':
+                            [('Formal languages', 'glossary-chomsky-hierarchy')]
+                    }
+                )
+            ),
+            ('some_processors_2.md',
+                KordacResult(
+                    html_string=self.read_test_file(self.test_name, 'some_processors_2_expected.html', strip=True),
+                    title='Another Example Title',
```

Comment:
Using the same title and some content overlaps with the last test. A little confusing when first looking at this test, I'm at argument over whether it should change as it is a perfectly valid example. Change only if you desire.